server/db-middleware/general-api.js

```diff
@@ -311,6 +312,16 @@ apiModule.getUniversity = function (filterConfig) {
     .select('-__v');
 };
 
+apiModule.getSkills = function (filterConfig) {
+  filterConfig = filterConfig || '';
```

Comment:
parameter reassignment is a bad practice
also it is restricted by our linter rules, haven't you seen this?